framework/src/main/java/org/checkerframework/framework/util/CheckerMain.java

```diff
@@ -27,7 +27,11 @@
  *   <li>add the {@code javac.jar} to the runtime classpath of the process that runs the Checker
  *       Framework.
  *   <li>add {@code jdk8.jar} to the compile time bootclasspath of the javac argument list passed to
- *       javac
+ *       javac while using Java 8
```

Comment:
Start the sentence with this information: "If using Java 8, ...".  Better, give this bullet point two sub-bullet points, one for Java 8 and one for Java 9+.

Reply:
Done